assets/js/blocks/collection-filters/inner-blocks/attribute-filter/components/placeholder.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, category, external } from '@wordpress/icons';
+import { getAdminLink } from '@woocommerce/settings';
+import { Placeholder, Button } from '@wordpress/components';
+
+export const AttributesPlaceholder = ( {
+	children,
+}: React.PropsWithChildren ) => (
```

Comment:
This utility type is for optional children, is children optional here? 

If children is mandatory and also you have no other props this could just be typed as:

```suggestion
}: { children: ReactNode } ) => (
```